app/controllers/subscribers_controller.rb

```diff
@@ -17,6 +17,24 @@ def create
     end
   end
 
+  def unsubscribe
```

Comment:
Could you please extract this action and the `cancel_subscription` to another controller? This one is turning a little cumbersome.